include/deal.II/lac/sparse_vanka.h

```diff
@@ -240,9 +244,9 @@ class SparseVanka
   /**
    * Apply the inverses corresponding to those degrees of freedom that have a
    * @p true value in @p dof_mask, to the @p src vector and move the result
-   * into @p dst. Actually, only values for allowed indices are written to @p
-   * dst, so the application of this function only does what is announced in
-   * the general documentation if the given mask sets all values to zero
+   * into @p dst. The transpose of the inverse is applied instead if @p transpose equals true.
+   * Actually, only values for allowed indices are written to @p dst, so the application of this function only does what is announced in
+   * the general documentation if the given mask sets all values to zero.
```

Comment:
Could you please break the lines at 80 characters? This will be required by the `indent` CI. You can also run `make indent` if you have clang-format installed according to https://github.com/dealii/dealii/wiki/Contributing (in case you haven't already).

Reply:
Due to this part of comments containing "@p", it is skipped from formatting by `make indent`. So, I manually edited the line to be less than 80 characters long.